roles/openshift_master_facts/tasks/main.yml

```diff
@@ -49,6 +49,7 @@
       registry_url: "{{ oreg_url | default(None) }}"
       oauth_grant_method: "{{ openshift_master_oauth_grant_method | default(None) }}"
       sdn_cluster_network_cidr: "{{ osm_cluster_network_cidr | default(None) }}"
+      external_ip_network_cidr: "{{ external_ip_network_cidr | default(None) }}"
```

Comment:
I think `openshift_master_external_ip_network_cidr` would line up with the other vars and we should also add this to the example inventories.